docs/modules/ROOT/pages/tutorials/getting-started.adoc

```diff
@@ -397,18 +389,15 @@ This should list 5 Pods, maybe still in `ContainerCreating`.
 kubectl -n syn get app root -o jsonpath="{.status.sync.status}"
 ----
 This command should return `Synced`.
-. Retrieve the admin password for Argo CD
+. Retrieve the admin password for Argo CD and create a port-forward to access it with a web browser:
 +
 [source,shell]
 ----
 kubectl -n syn get secret steward -o json | jq -r .data.token | base64 --decode
+kubectl -n syn port-forward pod/$(kubectl -n syn get pod -o custom-columns=:metadata.name | grep argocd-server) 8080:8080
 ----
-. Now you can access Argo CD by forwarding the port and opening it in your browser with http://localhost:8443. Login with the username `admin` and the password retrieved in the previous step.
 +
-[source,shell]
-----
-kubectl -n syn port-forward svc/argocd-server 8443:443
-----
+. Now you can access Argo CD by forwarding the port and opening it in your browser with http://localhost:8080. Login with the username `admin` and the password retrieved in the previous step.
```

Comment:
The port forward already happend at this point. All that is left, is opening it in the browser. Or did I miss something?